src/pages/signin/ChangeExpensifyLoginLink.js

```diff
@@ -24,7 +25,8 @@ const ChangeExpensifyLoginLink = ({credentials}) => (
         >
             <Text style={[styles.link]}>
                 Not&nbsp;
-                {credentials.login}
+                {Str.removeSMSDomain(credentials.login)}
```

Comment:
Heads up there there is an open issue for this

https://github.com/Expensify/Expensify.cash/issues/2323

Reply:
Ah, oops. Didn't think to look for open issues, just happened upon it in testing and figured it was a super easy fix.

Reply:
Should I remove this line so that the contributor who has already put in the work to fix it can still get paid?